lib/shoulda/matchers/active_record/have_db_index_matcher.rb

```diff
@@ -65,6 +65,56 @@ module ActiveRecord
       #
       # @return [HaveDbIndexMatcher]
       #
+      # has an index and an array of columns
```

Comment:
This was copied from above, needs to be changed to a wording that makes sense in this context.

Reply:
This section should probably also be moved to be above the first `Qualifiers` header. It doesn't really make sense to have this under the `Qualifiers` section, and I think the return statement (`@return [HaveDbIndexMatcher]`) is always supposed to be at the bottom of the documentation anyway.